.github/issue_template.md

```diff
@@ -0,0 +1,15 @@
+# Task description
+
+Describe what should be done.
+
+# Implementation suggestion
+
+Describe how it should be implemented.
+
+# Definition of done
+
+Describe when this task is considered finished.
+
+# Context
+
+Add additional context in the from of other issues/PRs/discussions.
```

Comment:
I wonder if we need as much content here - or if we need this file at all. I would imagine most people would just delete this without even reading when creating a new issue.

Reply:
We can't make people write better issues/bug-reports/feature requests. But at least we can nudge them. That is the point of having the templates. We can also just not add them and keep the current status quo if you don't like these things.

Reply:
Here, I was talking specifically about the "Open a blank issue" option. If the user selects it - do we really need to show the same exact fields as for "Task" option?

Reply:
delete the template